src/server/osd_window.rs

```diff
@@ -153,7 +153,7 @@ impl SwayosdWindow {
 		let max = brightness_backend.get_max() as f64;
 		let progress = self.build_progress_widget(brightness / max);
```

Comment:
This too?

Reply:
The same here... but I think I should `.round` this one too so it's consistent with actual label value shown.

Reply:
Sounds good to me 😄

Reply:
Actually, rounding this makes the progress disappear... which makes sense, because I'm rounding values in `[0-1]`. So, I won't change this.